src/main/src/main/java/org/geoserver/catalog/impl/CatalogImpl.java

```diff
@@ -141,6 +141,7 @@ public Iterable<CatalogValidator> getValidators() {
     }
 
     public void setFacade(CatalogFacade facade) {
+        rawFacade = facade;
```

Comment:
So the actual important change is here... the refactor of the event management obfuscates it (still a good thing, but it's a distraction when trying to find what makes the fix actually work).